bndtools.core/src/bndtools/editor/contents/GeneralInfoPart.java

```diff
@@ -111,6 +111,7 @@ public String getLabel() {
 
     private static final ServiceComponent SERVICE_COMPONENT_STAR = new ServiceComponent("*", new Attrs());
     private static final String DSANNOTATIONS_STAR = "*";
+    private static final String NO_DSANNOTATIONS_PATT = "!*";
```

Comment:
The no annotations should be empty string.

    -dsannotations:

is the way to disable ds annotations processing.